bin/parse

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
```

Comment:
this file shouldn't be in the `bin` directory. create a `utils` directory and move `parse` and `help` file to the `utils` directory.